easybuild/easyconfigs/n/NiBabel/NiBabel-2.3.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'NiBabel'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://nipy.github.io/nibabel'
+description = """NiBabel provides read/write access to some common medical and neuroimaging file formats,
+ including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT
+ as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and
+ annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Pillow', '5.3.0', versionsuffix),
+]
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_defaultclass = 'PythonPackage'
```

Comment:
exts_defaultclass should be removed, the PythonBundle takes care of this already.

Reply:
removed the exts_defaultclass